options/ansi/generic/string-stubs.cpp

```diff
@@ -455,6 +455,11 @@ char *strerror(int e) {
 	case ENOTUNIQ: s = "Name not unique on network (ENOTUNIQ)"; break;
 	case ERESTART: s = "Interrupted system call should be restarted (ERESTART)"; break;
 	case EUSERS: s = "Too many users (EUSERS)"; break;
+
+	#ifdef EIEIO
```

Comment:
Formatting issue, the #ifdef should not be indented here. Same for the #endif.

Reply:
Done